app/helpers/patients_helper.rb

```diff
@@ -73,4 +73,11 @@ def dcaf_pledge_limit_help_text
 
     "Pledge Limit Guidelines:<br />1st trimester (7-12 weeks): $#{first_tri}<br />2nd trimester (12-24 weeks): $#{second_tri}<br />Later care (25+ weeks): $#{later_care}"
   end
+
+  def duration_since_last_call
+    @patient = Patient.find(params[:id])
```

Comment:
Rails is smart/magic enough to infer the `patient` here, so you can ditch this DB call and just have the relation below be `calls.last.created_at`.